op-e2e/actions/proofs/l1_lookback_test.go

```diff
@@ -0,0 +1,157 @@
+package proofs
+
+import (
+	"testing"
+
+	actionsHelpers "github.com/ethereum-optimism/optimism/op-e2e/actions/helpers"
+	"github.com/ethereum-optimism/optimism/op-e2e/actions/proofs/helpers"
+	"github.com/ethereum-optimism/optimism/op-program/client/claim"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/core/types"
+	"github.com/stretchr/testify/require"
+)
+
+const numL2Blocks = 1 << 3
```

Comment:
Nit but adding arithmetic doesn't improve comprehension for the const. Would prefer the resulting whole number.